ArraysHomework/ArraysHomework.cs

```diff
@@ -23,26 +26,31 @@ public void Arrays()
             // Do not edit the loop that assert the passing case. Instead, add code above it to make it pass.
 
             // hint you need some form of repetition.
+
+            for (int i = 0; i < myArray.Length; i++)
+            {
+                myArray[i] = 10;
+            }
             for (int i = 0; i < myArray.Length; i++)        // Don't edit from HERE
             {                                               //
                 Assert.IsTrue(myArray[i] == 10);            // 
             }                                               // to HERE.
 
 
             // what wrong with this array declaration? Fix it and make it compile
-            string stringArray = new string[5];
+            string[] stringArray = new string[5];
 
             // What about this one? Fix it and make it compile
-            bool[] boolArray = new bool[];
+            bool[] boolArray = new bool[5];
 
             // What about this one? Fix it and make it compile
-            bool[] boolArray2 = new [20];
+            bool[] boolArray2 = new bool[20];
 
             // Take this array and copy its contents into a new array 
             char[] copyMe = { 'a', 'b', 'c' };
-            char[] copiedArray = new char[3];
+            char[] copiedArray = copyMe;
```

Comment:
This won't actually copy the array. `copyMe` and `copiedArray` are both references to the same array.